### PR TITLE
Blog on Design Tokens

### PR DESCRIPTION
**Pull request by lovin26**
Introduced a new blog on Design Tokens

### CODE REVIEW

File: website/blog/2023-10-18-Design-Tokens.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: "Design Tokens"
+author: Lovin Ahmed
+---
+---
+
+There is a lot of talk going on lately about design tokens. So, what are design tokens? As I understand it, design tokens are platform-agnostic variables that represent the look and feel of your product. Design tokens are the only source of truth for the design system.
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2023-10-18-Design-Tokens.md:7]: Remove the first sentence.

File: website/blog/2023-10-18-Design-Tokens.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+## What are Design Tokens?
+
+**"Design tokens constitute atomic building blocks of your design system that can be used to define "typography", "colors", and "spacing" and shared across platforms like iOS, Android, and regular websites. They provide the foundation for theming and design process automation."**
+
+## Who uses Design Tokens?
+
+One of the fundamental questions when creating a design system is:
+
+**“Who is going to use it and how?”**
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2023-10-18-Design-Tokens.md:21]: Change to H3 instead use '###'.

File: website/blog/2023-10-18-Design-Tokens.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+We will learn more about using design tokens with or without a design system in the coming blogs.
+
+
+## Conclusion
+
+In conclusion, design tokens are not just a novel concept; they are the game changer for digital product design.
+
+As we simplify the process, we can standardize the creation of design tokens, the naming conventions of design tokens, and the different types of design tokens. Using design tokens instead of hard coded values ensures more flexible design systems that are easy to build, maintain, and scale.
+
+In the next blog post, see how it works with SaaS applications.
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2023-10-18-Design-Tokens.md:152]: see how it works iwth Saas application and with WaveMaker.

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

**Comment by lovin26** [comment]
It looks good to me

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**